### PR TITLE
DO NOT MERGE: "single" variant

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
added new term single, PATO_0040021. Addresses #41

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
added new term flared, PATO_0040022. Removed synonym from splayed. addresses #57

**Comment by matentzn** [comment]
@nicolevasilevsky If you want this merged, you will have to resolve the conflicts with your previous edits!

**Comment by nicolevasilevsky** [comment]
ok, I think I resolved the conflicts
@matentzn

**Comment by nicolevasilevsky** [comment]
but now there is an issue with the build...

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -20359,6 +20358,32 @@ is_a: PATO:0000383 ! female
 property_value: http://purl.org/dc/elements/1.1/creator http://orcid.org/0000-0001-5208-3432
 property_value: http://purl.org/dc/elements/1.1/date 2018-11-16T22:52:49Z xsd:dateTime
 
+[Term]
+id: PATO:0040021
+name: single
+def: "A quality inhering in a bearer by virtue of the bearer's being present as one entity." [https://orcid.org/0000-0003-3162-7490]
```
Comment (matentzn) [inline src/ontology/pato-edit.obo]: i dont like this

Comment (matentzn) [reply src/ontology/pato-edit.obo]: i am sorry nicole.

**Comment by cmungall** [comment]
@nicolevasilevsky can you address the changes?

Travis now fixed on master

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo]: what don't you like, @matentzn? this term, the def, both? Do you not want to add it?

**Comment by nicolevasilevsky** [comment]
@cmungall should we wait for Nico to reply? He's on vacation at the moment

**Comment by matentzn** [comment]
@nicolevasilevsky it is a bit unclear to me what this trait "single" should be used for. Is this used in the sense of "count of one"? What is the use case for this term?

**Comment by nicolevasilevsky** [comment]
@wdahdul requested this - so she can best answer your question, @matentzn

**Comment by wdahdul** [comment]
@nicolevasilevsky @matentzn sorry, see my reply in  #215 (also copied here):

A better label for this is "unpaired" to distinguish from count=1. In Phensocape, we have phenotypes describing entities that are paired bilateral structures vs. unpaired single structures. (For example in some fish taxa the abdominal plates or scutes can be paired or unpaired). If 'unpaired' is added, we should add 'paired' for balance.

**Comment by matentzn** [comment]
Perfect that makes sense to me!

**Comment by nicolevasilevsky** [comment]
thanks @wdahdul, I added paired and unpaired on this PR

**Comment by matentzn** [comment]
@nicolevasilevsky you can address the changes, but hold on with merging it for now. I would like to migrate to ODK first, then I will merge this myself. But would be good if Clares comments were addressed by then!

**Comment by nicolevasilevsky** [comment]
@matentzn I just made the changes that @Clare72 requested. There were conflicts that I just fixed.

@Clare72 also said "Your term creation attribution is different for 'paired' - I think this is the correct one and the others need changing." - I don't know which created by annotation to use for PATO, should it be 'created_by' or dc:creator?

We are using 'created_by' for Mondo and HPO, this is what is saved in my Protege settings.

**Comment by matentzn** [comment]
Dont worry about this; @dosumis and I will deal with this next Week.

**Comment by nicolevasilevsky** [comment]
thank you @matentzn

**Pull request merged**